packages/api-explorer/src/components/SideNav/SideNavMethods.tsx

```diff
@@ -85,12 +85,16 @@ export const SideNavMethods = styled(
           {Object.values(methods).map((method) => (
             <li key={method.name}>
               <Link
-                to={`${buildMethodPath(
-                  specKey,
-                  tag,
-                  method.name,
-                  searchParams.toString()
-                )}`}
+                to={() => {
+                  // TODO: span behaving like link with custom navigate?
```

Comment:
Wanted to follow up with @josephaxisa in call to understand the approach for handling this.